retrofit/src/main/java/retrofit/OkHttpCall.java

```diff
@@ -151,7 +158,8 @@ private void callSuccess(Response<T> response) {
       // If the underlying source threw an exception, propagate that rather than indicating it was
       // a runtime exception.
       catchingBody.throwIfCaught();
-      throw e;
+      throw new RuntimeException("Problem has occurred during conversion of response for "
```

Comment:
Yeah, that's the second thing that I've mentioned in the PR description.

But but, otherwise we won't be able to include useful info into the exception. 

Though we can *try* to modify `exception`'s message via Reflection…